modules/ROOT/pages/migration-from-6x.adoc

```diff
@@ -3,62 +3,310 @@
+
+.Updated default text patterns
+[source, ts]
+----
+default: [
+  { start: '#', format: 'h1', trigger: 'space' },
+  { start: '##', format: 'h2', trigger: 'space' },
+  { start: '###', format: 'h3', trigger: 'space' },
+  { start: '####', format: 'h4', trigger: 'space' },
+  { start: '#####', format: 'h5', trigger: 'space' },
+  { start: '######', format: 'h6', trigger: 'space' },
+  { start: '1.', cmd: 'InsertOrderedList', trigger: 'space' },
+  { start: '*', cmd: 'InsertUnorderedList', trigger: 'space' },
+  { start: '-', cmd: 'InsertUnorderedList', trigger: 'space' },
+  { start: '>', cmd: 'mceBlockQuote', trigger: 'space' },
+  { start: '---', cmd: 'InsertHorizontalRule', trigger: 'space' },
+]
+----
+
+For information on the **text_patterns**, see xref:content-behavior-options.adoc#text_patterns[Text Patterns].
+
+=== Removed
+
+==== The autocompleter `ch` configuration property has been removed. Use the `trigger` property instead.
+// #TINY-8929
+
+Previously, the `ch` configuration property was used to specify the character that would trigger the autocompleter.
+
+This property was deprecated in link:https://www.tiny.cloud/docs/tinymce/6/6.2-release-notes/#deprecated[{productname} 6.2] and has been removed in {productname} 7.0. Instead, use the `trigger` property to specify the `string` that will trigger the autocompleter.
+
+If `+editor.ui.registry.addAutocompleter(name, options)+` was used in your configuration, updating your configuration from `ch: '<string>',` to `trigger: '<string>',` is required.
+
+NOTE: The new `trigger` option can handle multiple character strings as the trigger.
+
+For more information, visit the updated xref:autocompleter.adoc[Autocompleter] documentation.
+
 
 [[commands-and-queries]]
 == Commands and queries
 
-...
+=== Changed
+
+==== Removed `InsertOrderedList` and `InsertUnorderedList` commands from the core.
+
+Previously, native list commands were executable through various text patterns, leading to undefined browser behavior due to the embedded list code.
+
+As of {productname} 7.0, these commands have been extracted from the core list commands.
+
+[NOTE]
+To utilize these commands, include `plugins: [ 'lists' ]` in the configuration. This action enables the commands previously removed from the core, providing access through the `lists` plugin.
 
 [[options]]
 == Options
 
-...
+=== Changed
+
+[[highlight-on-focus]]
+==== `highlight_on_focus`
+
+Previously, the default value for the editor configuration option `highlight_on_focus` was set to `false` by default. In {productname} 7.0 this option is now by default set to `true`
+
+As a result, the focus outline for the editor will by default be displayed when the focus is on the editor.
+
+Customers who have created their own custom skins, or have implemented their own outline and wish to not to use the default one will need to add the `highlight_on_focus: false` option to their {productname} configuration.
```

Comment:
This might also implies that the customers who wish to provide custom CSS for the outline needs to set the option to `false`, which disables showing the outline on focus